ext/zip/zip_stream.c

```diff
@@ -234,6 +234,14 @@ php_stream *php_stream_zip_open(const char *filename, const char *path, const ch
 			return NULL;
 		}
 
+		if (password != NULL) {
+			if (zip_set_default_password(stream_za, password) != 0) {
```

Comment:
Wouldn't it make more sense to use zip_fopen_encrypted, rather than using zip_set_default_password for a single file?

Reply:
`zip_fopen_encrypted()` is only available as of libzip 1.0.0, but even "master" still supports libzip ≥ 0.11. :(

Reply:
I see, then this implementation makes sense...